### PR TITLE
Ensure that completed status is selected by default on search contrib…

### PR DESCRIPTION
**Pull request by seamuslee001**
…ution form

Overview
----------------------------------------
This fixes a small bug where when loading the contribution form the status field doesn't have a default of completed status.

Before
----------------------------------------
contribution_search_change_pre

After
----------------------------------------
contribution_search_change_post

ping @eileenmcnaughton @monishdeb @mattwire

### CODE REVIEW

File: CRM/Contribute/Form/Search.php [hunk 1]
```diff
@@ -153,8 +153,8 @@ public function preProcess() {
    */
   public function setDefaultValues() {
     $this->_defaults = parent::setDefaultValues();
-    if (empty($this->_defaults['contribution_status'])) {
-      $this->_defaults['contribution_status'][1] = 1;
+    if (empty($this->_defaults['contribution_status_id'])) {
+      $this->_defaults['contribution_status_id'][1] = 1;
```
Comment (pradpnayak) [inline CRM/Contribute/Form/Search.php]: Hardcoded?
```
$this->_defaults['contribution_status_id'][1] = CRM_Core_PseudoConstant::getKey(
  'CRM_Contribute_BAO_Contribution',
  'contribution_status_id',
  'Completed'
);
```

Comment (seamuslee001) [reply CRM/Contribute/Form/Search.php]: Thanks @pradpnayak good point have changed to the PseudoConstant

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by eileenmcnaughton** [comment]
@seamuslee001 I guess that makes sense but note that - https://github.com/civicrm/civicrm-core/pull/14624 changes it to not kick in under force

**Comment by seamuslee001** [comment]
@eileenmcnaughton i think that is ok but i would appreciate others thoughts re the force side of things

**Comment by eileenmcnaughton** [comment]
@seamuslee001 from my testing the force is keeping the old behaviour - ie try going from manage contribution page to the contributions from that page this month in an older version - I think it is not enforcing a status

**Pull request merged**